src/plugins/cordova-plugin-camera/sim-host-panels.html

```diff
@@ -1,6 +1,6 @@
 <!-- Copyright (c) Microsoft Corporation. All rights reserved. -->
 
-<cordova-panel id="camera" caption="Camera" data-loc-id="553fc9a5">
+<cordova-panel id="camera" caption="Camera" panel-label="Camera Panel" data-loc-id="553fc9a5">
```

Comment:
It seems `panel-label` and `read-label` mean basically the same thing. For consistency we should replace them with a single attribute that is used for both. Perhaps `spoken-text` or something like that. 

If we *do* use `spoken-text`, then we should change `read-out` to something consistent (I think `spoken` and related words is clearer than `read`) - `read-out` could just become `spoken`.